plugins/modules/rma_workflow_manager.py

```diff
@@ -5,7 +5,7 @@
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
-__author__ = ("Trupti A Shetty, Mohamed Rafeek, Madhan Sankaranarayanan")
+__author__ = ("Trupti A Shetty, Ajith Andrew J, Mohamed Rafeek, Madhan Sankaranarayanan")
```

Comment:
Please append the author names instead of inserting them.

Reply:
Ok Madhan. Updated correctly